src/components/WorkflowsExplorer/Workflow/WorkflowHistoryTable.tsx

```diff
@@ -31,25 +27,27 @@ const WorkflowHistoryTable = (props : {workflow: string}) => {
                     <thead>
                         <tr>
                             <th>Run ID</th>
-                            <th>Attempt number</th>
-                            <th>Started</th>
-                            <th>Actions</th>
+                            <th>Attempt ID</th>
+                            <th>Run Start Time</th>
+                            <th>Duration</th>
+                            <th>Status</th>
                         </tr>
                     </thead>
                     <tbody>
-                        {runs.map((run : {id: number, run: StateFile}) => (
+                        {data[0].runs.map((run: any) => (
```

Comment:
Why just the first entry of the array? What are the other entries?

Reply:
If I'm correct, the new "useFetchWorkflow" hook returns a response (from the "fetch-then" javascript api) as an array of size 1 containing only the object we are actually interested in.